.prettierrc.cjs

```diff
@@ -6,7 +6,7 @@ module.exports = {
   tabWidth: 2,
   trailingComma: 'es5',
   useTabs: true,
-  plugins: ['./node_modules/prettier-plugin-astro'],
+  plugins: ['./node_modules/prettier-plugin-astro/dist/index.js'],
```

Comment:
I think we can also simply specify `plugins: ['prettier-plugin-astro']` in Prettier 3

Reply:
Yes, I would recommend to do this actually, it's better